deps.edn

```diff
@@ -114,6 +114,7 @@
   org.clojure/math.combinatorics            {:mvn/version "0.1.6"}              ; combinatorics functions
   org.clojure/math.numeric-tower            {:mvn/version "0.0.5"}              ; math functions like `ceil`
   org.clojure/tools.logging                 {:mvn/version "1.2.4"}              ; logging framework
+  org.clojure/tools.macro                   {:mvn/version "0.1.5"}              ; local macros
```

Comment:
I can't find anywhere where you're using this

Reply:
i had it in one part of the change but its no longer there. This is however a transitive dep brought in by compojure so we should leave it in. We use it in a few places

```shell
compojure/compojure 1.6.2
  . org.clojure/tools.macro 0.1.5
```